### PR TITLE
Chore/separate build processes

### PR DESCRIPTION
**Pull request by bob-voorhoede**

### CODE REVIEW

File: projects/browser-extension/vue-app/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+> 1%
+last 2 versions
+not dead
```
Comment (vvandijck) [inline projects/browser-extension/vue-app/.browserslistrc:3]: Is this a legitimate value?

File: projects/browser-extension/vue-app/src/views/apple-ios.view.vue [hunk 2]
```diff
@@ -17,13 +17,16 @@
         No touch icons detected.
       </p>
       <properties-list>
-        <template v-for="icon in touchIcons">
-          <dt :key="`${icon.url}-key`">
```
Comment (vvandijck) [inline projects/browser-extension/vue-app/src/views/apple-ios.view.vue:21]: I did this, moving the `key` to the `<template>`, already for all views in the web-app 👍

File: projects/browser-extension/vue-app/src/views/facebook.view.vue [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <template>
   <div class="facebook">
-    <switch-buttons
+    <!-- <switch-buttons
```
Comment (vvandijck) [inline projects/browser-extension/vue-app/src/views/facebook.view.vue:3]: Is this documented somewhere? Maybe put it in Trello.

**Review by vvandijck — APPROVED** [review]

File: projects/browser-extension/vue-app/.browserslistrc (hunk 1 above)
Comment (bob-voorhoede) [reply projects/browser-extension/vue-app/.browserslistrc:3]: Comes from vue-cli install. Not really necessary in the context of a browser-extension, could be just chrome and firefox. But there's more pressing issues to fix first ;)

File: projects/browser-extension/vue-app/src/views/facebook.view.vue (hunk 3 above)
Comment (bob-voorhoede) [reply projects/browser-extension/vue-app/src/views/facebook.view.vue:3]: It's in Trello ;)

**Pull request merged**